packages/avs-bot/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "avs-bot",
```

Comment:
Please change to `@wireapp/bot-avs`.

Reply:
https://github.com/wireapp/wire-web-packages/pull/3070/commits/7efc72ee040d93ae85446687ed22c2894cb3f08a